xml/depl_nodes.xml

```diff
@@ -3195,6 +3195,30 @@ getent group postgres | cut -d: -f3</screen>
     </listitem>
    </varlistentry>
    <varlistentry>
+    <term><guimenu>
+      Scheduler Options: Virtual Disk to Physical Disk allocation ratio
+     </guimenu>
+    </term>
+    <listitem>
+     <para>
+      Set the <quote>overcommit ratio</quote> for virtual disks for &vmguest;s on
+      the &compnode;s. A ratio of <literal>1.0</literal> means no
+      overcommitment.<remark condition="accuracy">2017-01-02 - dpopov: DEVS FIXME Check whether this is correct.</remark>
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term><guimenu>
+      Scheduler Options: Reserved Memory for Nova Compute hosts (MB)
+     </guimenu>
+    </term>
+    <listitem>
+     <para>
+      Amount of memory reserved for Nova Compute hosts.<remark condition="accuracy">2017-01-02 - dpopov: DEVS FIXME Check whether this is correct.</remark>
```

Comment:
its the reverse, amount of Host memory reserved and not used for allocating VMs by nova-compute